### ISSUE TITLE
User is not able to drag the "AzureFun.js" grid section using single pointer mode of operation: A11y_Azure Tools for VS Code_Azure Function_Default Screen_DraggingMovement. [Win]

### ISSUE THREAD

**Issue by keerthiduvvuri** [microsoft/vscode-azurefunctions#4100]
[[Check out Accessibility Insights!](https://accessibilityinsights.io/)]- Identify accessibility bugs before check-in and make bug fixing faster and easier.

### GitHubTags:
#A11yWCAG2.2; #A11yTCS; #A11ySev3; #Azure Tools for VS Code; #SH_AzureToolsforVSCode_Win32_April2024; #DesktopApp; #Win11; #WCAG2.5.7; #Dragging Movements; #A11yMAS; #FTP;.

### Environment and OS details:
Visual Studio Code for Win32 version 1.88.1
OS Version: Windows 11 Enterprise 23H2

### Repro Steps:

1. Open Visual studio code.
2. Click on extension button and search for "Azure Functions".
3. Click on install and move to the "Azure" button.
4. Observe that User is not able to resize "AzureFun.js" grid section using single pointer mode of operation.

### Actual Result:
User is not able to drag the "AzureFun.js" grid section using single pointer mode of operation.
### Expected Result:
User should be able to drag the "AzureFun.js" grid section using single pointer mode of operation without dragging.

### User Impact:
Functionality provided only with dragging will create difficulty for users using input devices like a trackball, head pointer, eye-gaze system to perform the operation.

### Attachment:
[Video 4-23 at 15.16.webm](https://github.com/microsoft/vscode-azurefunctions/assets/98728920/a56ee160-ac48-48df-8233-f431ea8a86dc)

[Video 4-23 at 15.16 (1).webm](https://github.com/microsoft/vscode-azurefunctions/assets/98728920/6667197b-4df3-4857-9530-1674bdb24759)

**Comment by Yash14j** [comment]
GithubTags:#Rev:yaja;
